buddyboss/buddyboss-platform
# Fixed: "and" string unable translate in the login page

### All Submissions:






### Changes proposed in this Pull Request:



Fixes https://github.com/buddyboss/buddyboss-platform/issues/2455 .

### How to test the changes in this Pull Request:

1. Set **Terms of Service** and **Privacy Policy** link on the login page
2. Go to **Login** page
3. See both the links after the Login button and these concerted with the "and" string
4. It is working when translating it with any other language using the loco translation plugin

### Proof Screenshots or Video
https://www.loom.com/share/3ddc80b0adbd41bdaa89f59d0eabb51e



### Other information:




### Changelog entry

> Enter a summary of all changes on this Pull Request. This will appear in the changelog if accepted.
